association.html

```diff
@@ -1,3 +1,5 @@
+---
```

Comment:
I don't think we need to specify a layout.

On Fri, Apr 24, 2015 at 8:59 AM btsibr <notifications@github.com> wrote:

> In association.html
> <https://github.com/pinewoodgreens/pinewoodgreens.github.io/pull/1#discussion_r29045862>
> :
>
> > @@ -1,3 +1,5 @@
> > +---
>
> Refreshing my memory, do we need to specify a layout or is there a default?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pinewoodgreens/pinewoodgreens.github.io/pull/1/files#r29045862>
> .
>